packages/main/src/Input.js

```diff
@@ -224,6 +224,19 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 *
+		 * Defines whether the value is shown in the input.
+		 *
+		 * @type {Boolean}
+		 * @defaultvalue false
+		 * @since 1.0.0-rc.9
+		 * @private
+		 */
+		_hideValue: {
```

Comment:
This API seems strange: when the new property is set, the value is effectively "". I don't object so strongly, but won't it be easier not to change the Input.js at all, and then from DateRangePicker set the input value to "" and then back to whatever it needs to be when the user selects a date.

It's just that Input.js is very central and we should try not to overcomplicate its logic unless really necessary.

Reply:
I still think we should not have this property in Input.js. This is logic specific to the date range picker. If the whole control becomes unusable when the input has no value, then we should change its internal logic to store the first/second clicked dates in internal variables and check for these instead.